plugins/vendor/services/ZoomVendorService.php

```diff
@@ -0,0 +1,431 @@
+<?php
+/**
+ * @service zoomVendor
+ * @package plugins.vendor
+ * @subpackage api.services
+ */
+class ZoomVendorService extends KalturaBaseService

```

Comment:
I don't see you are applying the partner filter on the criteria this is why you are always doing:
$partnerId = kCurrentContext::getCurrentPartnerId();
		$zoomIntegration = VendorIntegrationPeer::retrieveSingleVendorPerPartner($accountId,
			VendorTypeEnum::ZOOM_ACCOUNT, $partnerId);

Is there a reason for this?

Reply:
done